tests/AerospikeCacheTest.php

```diff
@@ -3,7 +3,200 @@
 
 namespace Lmc\AerospikeCache;
 
-class AerospikeCacheTest
+use PHPUnit\Framework\TestCase;
+
+class AerospikeCacheTest extends TestCase
 {
+    /**
+     * @dataProvider provideStatusCodes
+     */
+    public function testShouldConfirmItemPresence(int $statusCode, bool $expectedReturnedValue): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeKey = ['ns' => 'test', 'set' => 'cache', 'key' => 'foo'];
+        $aerospikeMock->expects($this->once())
+            ->method('initKey')
+            ->willReturn($aerospikeKey);
+
+        $aerospikeMock->expects($this->once())
+            ->method('get')
+            ->with($aerospikeKey)
+            ->willReturn($statusCode);
+
+        $hasItem = $aerospikeCache->hasItem('foo');
+
+        $this->assertSame($expectedReturnedValue, $hasItem);
+    }
+
+    /**
+     * @dataProvider provideStatusCodes
+     */
+    public function testShouldSaveItem(int $aerospikeStatusCode, bool $expectedSaveSuccessful): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeMock->method('put')
+                ->willReturn($aerospikeStatusCode);
+
+        $aerospikeMock->method('get')
+                ->willReturn(\Aerospike::OK);
+
+        $aerospikeKey = ['ns' => 'test', 'set' => 'cache', 'key' => 'foo'];
+        $aerospikeMock->method('initKey')
+                ->willReturn($aerospikeKey);
+
+        $cacheItem = $aerospikeCache->getItem('foo');
+        $saveSuccessful = $aerospikeCache->save($cacheItem);
+
+        $this->assertSame($expectedSaveSuccessful, $saveSuccessful);
+    }
+
+    /**
+     * @dataProvider provideStatusCodes
+     */
+    public function testShouldClearWithEmptyNamespaceName(int $aerospikeStatusCode, bool $expectedClearSuccessful): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeMock->method('truncate')
+            ->willReturn($aerospikeStatusCode);
+
+        $clearSuccessful = $aerospikeCache->clear();
+
+        $this->assertSame($expectedClearSuccessful, $clearSuccessful);
+    }
+
+    public function doClearWithEmptyNamespaceProvider(): array
```

Comment:
This provider also have prefix `provide...`.
And please keep the provided values on single lines, it is confusing like this, because it looks like a single value to me.

Reply:
Unused provider. I wil remove it